cmake/FindNodejs.cmake

```diff
@@ -62,6 +62,7 @@ set(nodejs_known_names
     "libnode.so.102" "libnode.102.dylib"
     "libnode.so.108" "libnode.108.dylib"
     "libnode.so.111" "libnode.111.dylib"
+    "libnode.so.115" "libnode.115.dylib"
```

Comment:
I wish there was some better way to do this in cmake so we didn't have to update it every time a new version of Node is released, but `find_library` doesn't support wildcards or patterns in any way.

Reply:
It's interesting that the specific versions are needed at all, why is this? I would have assumed that since we're building on top of Node, its development stuff needs to be installed, which would imply `libnode.so` / `libnode.dylib` will catch. Is that not the case on some platforms?

Reply:
I had wanted to make the comment that it's only me running into this probably because I build node from source and a standard `make install` doesn't put a `.so` link in place.